### PR TITLE
Feature/35204/newcastle importer

### PR DESCRIPTION
**Pull request by lauramccluskey1**
Functionality to add test status of 10 to benign, likely benign and non-pathological variants. Unittests have been updated and run ok for the Newcastle importer. Couple of integration issues when running all unittests- these have been flagged to @shilpigoeldev. Variant counts QA has been completed and checked.

### CODE REVIEW

File: test/lib/import/brca/providers/newcastle/newcastle_handler_test.rb [hunk 1]
```diff
@@ -203,7 +203,43 @@ def setup
...
+    noscope_genotype = Import::Brca::Core::GenotypeBrca.new(noscope_genemutation_record)
+    @handler.process_test_scope(noscope_genotype, noscope_genemutation_record)
+    @handler.process_test_status(noscope_genotype, noscope_genemutation_record)
+    genotypes = @handler.process_variant_records(noscope_genotype, noscope_genemutation_record)
+    assert_equal 'Unable to assign BRCA genetictestscope', noscope_genotype.attribute_map['genetictestscope']
+    assert_equal 10, genotypes[0].attribute_map['teststatus']
+  end
+
+  test 'process_noscope_with_benign_gene_mutation' do
+    noscope_genemutation_record = build_raw_record('pseudo_id1' => 'bob')
```
Comment (shilpigoeldev) [inline test/lib/import/brca/providers/newcastle/newcastle_handler_test.rb:229]: Can you keep the name for records align with their attribute like done in colorectal tests -  benign_record, likely_benign_record , thanks

**Review by shilpigoeldev — CHANGES_REQUESTED** [review]
Can you look into rubocop fixes for lib/import/helpers/brca/providers/rtd/rtd_constants.rb

**Comment by lauramccluskey1** [comment]
Changes requested in code review have now been made

**Review by shilpigoeldev — APPROVED** [review]
LGTM

**Pull request merged**